helix-core/src/main/java/org/apache/helix/controller/rebalancer/waged/WagedRebalancer.java

```diff
@@ -134,10 +145,9 @@ public void close() {
    */
   public Map<String, IdealState> computeNewIdealStates(ResourceControllerDataProvider clusterData,
       Map<String, Resource> resourceMap, final CurrentStateOutput currentStateOutput)
-      throws HelixRebalanceException {
+      throws HelixRebalanceException, JMException {
```

Comment:
Don't throw the monitor exception. Just log the error and ignore it. We don't want the monitor logic or bug to block business logic.